source/sas_compress.cpp

```diff
@@ -83,9 +83,6 @@ class LZ4Compressor : public SAS::Compressor
   static SAS::Compressor* get();
 
 private:
-  // The maximum size of input we can process in one go.
-  static const int MAX_INPUT_SIZE = 4096;
```

Comment:
Sure. Well then I guess the question is - is there any reason we initially try 4096? And if so, we should keep this line (but change the comment and variable name).